pennylane/devices/qutrit_mixed/sampling.py

```diff
@@ -0,0 +1,334 @@
+# Copyright 2018-2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Code relevant for sampling a qutrit mixed state.
```

Comment:
@Gabriel-Bottrill For clarity and reviewer ease, could you please provide a brief summary of the major differences between this file and the analogous one for the qubit device?

Reply:
For sure!